### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by JuliaRechkunova**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -40,6 +40,7 @@
 	* [Post-build action API](#post-build-action-api)
 	* [Browserify transformation and plugin API](#browserify-transformation-and-plugin-api)
 	* [List of officially supported plugins](#list-of-officially-supported-plugins)
+* [List of ready to use plugins and extensions](#plugins-and-extensions)
```
Comment (pragmadash) [inline docs/index.md]: ???

File: docs/index.md [hunk 2]
```diff
@@ -1117,12 +1118,23 @@ locator.registerInstance('browserifyPlugin', {
 );
 ```
 
+# Plugins and Extensions
+
 ## List of officially supported plugins
 
+* [catberry-handlebars](https://github.com/catberry/catberry-handlebars) – Catberry adapter for Handlebars template engine
+* [catberry-jade](https://github.com/catberry/catberry-jade) – Catberry adapter for Jade template engine
```
Comment (pragmadash) [inline docs/index.md]: Add catberry-dust.

File: docs/index.md [hunk 3]
```diff
@@ -1117,12 +1118,23 @@ locator.registerInstance('browserifyPlugin', {
...
 * [catberry-assets](https://github.com/catberry/catberry-assets) – The plugin that builds assets for each component using Gulp
 * [catberry-l10n](https://github.com/catberry/catberry-l10n) – The localization plugin
+* [catberry-l10n-handlebars-helper](https://github.com/catberry/catberry-l10n-handlebars-helper) – Handlebars helper for Catberry localization plugin
```
Comment (pragmadash) [inline docs/index.md]: Add catberry-l10n-dust-helper.

File: docs/index.md [hunk 4]
```diff
@@ -1117,12 +1118,23 @@ locator.registerInstance('browserifyPlugin', {
...
 * [catberry-uhr](https://github.com/catberry/catberry-uhr) – The universal HTTP(S) request implementation
+* [catberry-uri](https://github.com/catberry/catberry-uri) – URI parser implementation that has been developed strictly according to RFC 3986
```
Comment (pragmadash) [inline docs/index.md]: This is not a plugin.

**Comment by JuliaRechkunova** [comment]
@pragmadash Updated

**Pull request merged**